gulpfile.ts

```diff
@@ -338,6 +338,7 @@ task('less', function () {
 task('babel', ['concat-develop'], function () {
     return gulp.src(bundlePath)
         .pipe(babel({
+            "compact": true,
```

Comment:
We use uglifyjs for production build, this only for develop.

Reply:
How do I disable uglifyjs to run the final application for the user? I want to deploy the final application.

Reply:
How do I disable gulpfile to run the final application for the user? I want to deploy the final application.